Update Health runtime to 48>50 and more

yakushabb opened the pull request:
- Update runtime to 50
- Drop blueprint-compiler that is already provided by the runtime
- Use llvm20, as llvm19 is not supported for this runtime version
- Drop ineffective cleanup commands

yakushabb commented:
@Cogitri I've updated Health runtime to 49 with other small changes. Could you review the changes. Thanks.

Cogitri commented:
Hej, thanks for the PR, I'll take a look at it in the coming days :) Merry Christmas!

yakushabb committed:
Update runtime to 50

- Update runtime to 50
- Drop blueprint-compiler that is already provided by the runtime
- Use llvm20, as llvm19 is not supported for this runtime version
- Drop ineffective cleanup commands

yakushabb commented:
@Cogitri, I've updated runtime version to 50.

yakushabb commented:
@Cogitri, the runtime version 48 reached end-of-life. Would you review this PR? Thanks.
